CONTRIBUTING.md

```
@@ -1,181 +1,177 @@
-
-Join our Discord [here](https://chat.odysee.com/).
+After deciding what to work on, you can fork this repository, make your changes,
+and submit a pull request.
+  - A contributor wanting to reserve an issue in advance can leave a comment
+    saying that they're working on it. Contributors should respect other people's
+    efforts to complete issues in a timely manner and, therefore, not begin working
+    on anything reserved (or updated) within the last 3 days. If someone has been
+    officially assigned an issue via GitHub's assignment system, it is also not
+    available. Contributors are encouraged to ask if they have any questions about
+    issue availability.
+- Once the pull request is visible, a team member will review it and make sure
```

markdownlint-cli2 (0.23.2)

[warning] Inconsistent indentation for list items at the same level
Expected: 2; Actual: 0

(MD005, list-indent)